src/class-utils.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Parse.ly utilities class
+ *
+ * @package Parsely
+ * @since 3.2.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
+
+use WP_Post;
+
+/**
+ * Utility methods to be reused across the codebase.
+ *
+ * @since 3.2.0
+ */
+class Utils {
```

Comment:
> This PR also creates a utilities class to place reusable methods there.

Personally, I vote we avoid having a class called something generic like `Utils` to avoid it becoming catch-all file. If it's important enough to become a named function, we should probably be able to come up with a grouping for it.

e.g.

```suggestion
class Parsely_Dashboard_Links {
```

...or similar.

Reply:
I don't fully agree with this. I'm not sure we'll have more functions related to generate dashboard links in the near future, so a full-blown class for this kind of functions seems overkill.

The other option I considered was to simply put it in `class-parsely.php`, which has effectively become a catch-all file. When we tackle splitting that file, we might want to reconsider this.